doc/api/net.md

```diff
@@ -7,35 +7,53 @@ functions for creating both servers and clients (called streams). You can includ
 this module with `require('net');`.
 
 ## Class: net.Server
+<!-- YAML
+added: v0.7.5
```

Comment:
Are you sure? The docs change in which it was introduced seems to be in 499c615c9e515